tsconfig.json

```diff
@@ -15,8 +16,10 @@
       "@lib/*": ["lib/*"],
       "@domain/*": ["domain/*"],
       "@repository/*": ["repository/*"]
-    }
+    },
+    "types": ["vitest/globals"]
```

Comment:
maybe it's better to add a separate tsconfig for tests?

Reply:
this line should be removed, so, there's no need in separate tsconfig